grease-cli/src/Grease/Main.hs

```diff
@@ -941,6 +940,13 @@ simulateMacawSyntax la halloc archCtx simOpts parserHooks = do
         forM_ (Map.elems cfgs) $ \entrypointCfgs ->
           forM_ (startupOvForwardDecs <$> entrypointStartupOv entrypointCfgs) $ \startupOvFwdDecs ->
             Macaw.registerMacawOvForwardDeclarations bak funOvs startupOvFwdDecs
+
+        -- Register defined functions.
+        forM_ (CSyn.parsedProgCFGs prog) $ \(C.Reg.AnyCFG defCfg) -> do
+          C.SomeCFG defSsa <- pure $ C.toSSA defCfg
+          -- This could probably be a helper defined in Crucible...
+          let bindCfg c = C.bindFnHandle (C.cfgHandle c) (C.UseCFG c (C.postdomInfo c))
+          bindCfg defSsa
```

Comment:
This code isn't quite right, since really we should be checking if each function has a corresponding function override and doing something different if an override exists. I briefly looked into this, but it ended up being more involved than I expected, partly because we would have to change `stubs` in order to make this possible. See #178.

I don't particularly need the ability to override functions defined in an S-expression program just yet, so I am going to accept this deficiency for now, leaving #178 as future work.